libc/benchmarks/gpu/CMakeLists.txt

```diff
@@ -6,10 +6,15 @@ function(add_benchmark benchmark_name)
   cmake_parse_arguments(
     "BENCHMARK"
     "" # Optional arguments
-    "" # Single value arguments
+    "NUM_THREADS" # Single value arguments
     "LINK_LIBRARIES" # Multi-value arguments
     ${ARGN}
   )
+
+  if(NOT BENCHMARK_NUM_THREADS)
+    set(BENCHMARK_NUM_THREADS 1)
+  endif()
+  
```

Comment:
```suggestion
 
```

Reply:
We shouldn't need to pass a default, if no `LOADER_ARGS` are passed in then it will just be 1.